examples/cryptocurrency-advanced/backend/src/schema.rs

```diff
@@ -36,6 +36,8 @@ pub(crate) struct SchemaImpl<T: Access> {
     pub public: Schema<T>,
     /// History for specific wallets.
     pub wallet_history: Group<T, Address, ProofListIndex<T::Base, Hash>>,
+    /// Configuration of the service.
+    pub config: Entry<T::Base, Config>,
```

Comment:
Is config present after a data migration? Seemingly not, which may lead to panics during wallet creation.

Reply:
The configuration is being available only after migration to 0.2.0 version. Simple version of the cryptocurrency doesn't have a configuration.